jimpass/util.py

```diff
@@ -43,6 +43,21 @@ def srun(input_cmd: str, stdin: str = None, no_output: bool = False) -> (int, st
         res = run(input_cmd, stdout=DEVNULL if no_output else PIPE, stderr=DEVNULL, encoding="utf-8", shell=True)
         return res.returncode, res.stdout
 
+def wofi(mode: str = "dmenu", prompt: str = None, options: list = None,
```

Comment:
This is awfully similar to the rofi call. Since you manipulate a `runner` configuration parameter (which I think is a good idea), maybe it could just be added to the list of arguments in the function.
Naturally, we would have to come up with a new name for this function :)